Gems/Atom/RHI/DX12/Code/Source/RHI/StreamingImagePool.h

```diff
@@ -65,10 +68,26 @@ namespace AZ
             // Packed mips occupy a dedicated set of tiles.
             void AllocatePackedImageTiles(Image& image);
 
-            RHI::Ptr<ID3D12Heap> m_heap;
+            // Get the data reference of device heap memory usage 
```

Comment:
Get information related to device heap..

Reply:
It's actually for get the reference of the usage object so others can update the data.